### ISSUE TITLE
Additional parameter to configure the view

### ISSUE THREAD

**Issue by JBPressac** [drkane/datasette-reconcile#20]
Hello,
As for the [Wikidata reconcile API](https://wikidata.reconci.link/en/api), the view template for entities could not have the same domain as the reconciliation API (OpenRefine displays Wikidata candidates with an URL as https://www.wikidata.org/wiki/{{id}}). Would it be possible to add a new plugin configuration item to eventually replace the datasette instance URL (as specified below) for the "view" by a combination of a domain given as parameter and the id_field.

https://github.com/drkane/datasette-reconcile/blob/d71fcade9f5e106e2ee2f39fe7586df6cd7eec6c/datasette_reconcile/reconcile.py#L97

To be more concise, would it be possible to modify the domain of the URLs of the candidates:

![2021-05-28 18_32_04-Diaz Levriou troet xlsx - OpenRefine](https://user-images.githubusercontent.com/6371750/120015112-373a9680-bfe3-11eb-9d1a-a9b44d57806c.png)
 

Thanks,

**Comment by JBPressac** [comment]
Hello @drkane 
I tried several times to pull a request fir this issue but each time the checks fail during linting process. I also applied linting with Black and isort locally to avoid this problem, but this does not prevent GitHub to apply linting during check process. If you have any idea of the causes of this...

By the way, I do not understand the following command to install dependencies and tests: 

`pip install -e '.[test]'`

I successfully tried (on Microsoft Windows):

`pip install -e " C:\Users\jean-\Documents\GitHub\datasette-reconcile\"`

Where  C:\Users\jean-\Documents\GitHub\datasette-reconcile\ is the path to my local datasette-reconcile Git repository. But I don't understand what thoses brackets mean. And I am wondering if you did not mean "tests" instead of "test", referring to the tests folder ? 

Thank you,
